components/src/main/res/layout/component_editable_list_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_marginStart="@dimen/hmrc_spacing_16"
+    android:layout_marginEnd="@dimen/hmrc_spacing_16"
```

Comment:
I don't think we want these margins.
One reason is because usually we want the ripple of the secondary/icon buttons to stretch to the sides of the view, and so having the margin on the parent layout cuts into the ripple. The button has built in margins that mean the content is already spaced properly from the sides.
But also, this component will be added to an existing card, so if we don't have any padding it leaves it more flexible to fit into other layouts, which can add the padding as needed.